App bootstrapping fails

Using `ignite-cli@3.1.3`

```
ignite new TheApp                 
-----------------------------------------------
  (                  )   (                   
  )\ )   (        ( /(   )\ )    *   )       
 (()/(   )\ )     )\()) (()/(  ` )  /(   (   
  /(_)) (()/(    ((_)\   /(_))  ( )(_))  )\  
 (_))    /(_))_   _((_) (_))   (_(_())  ((_) 
 |_ _|  (_)) __| | \| | |_ _|  |_   _|  | __|
  | |     | (_ | | .` |  | |     | |    | _| 
 |___|     \___| |_|\_| |___|    |_|    |___|
-----------------------------------------------

An unfair headstart for your React Native apps.
https://infinite.red/ignite

-----------------------------------------------

🔥 igniting app TheApp
✔ Which boilerplate would you like to use? · ignite-bowser
✔ using the Infinite Red boilerplate v3 (code name 'Bowser')
✔ added React Native 0.59.3 in 20.64s
✔ Would you like to include Detox end-to-end tests? (y/N) · false
✖ ignite-bowser was not able to be installed. Is it a valid NPM module?
----------
Command failed: yarn add ignite-bowser --dev
warning @storybook/react-native > @storybook/core > autoprefixer > browserslist@2.11.3: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > autoprefixer > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-merge-rules > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-merge-rules > caniuse-api > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-minify-selectors > postcss-selector-parser > flatten@1.0.2: I wrote this module a very long time ago; you should use something else.
warning solidarity > gluegun > enquirer > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.
warning solidarity > gluegun > prompt-autocompletion > prompt-choices > set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.
warning solidarity > gluegun > enquirer > prompt-question > prompt-choices > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.
warning "react-navigation > react-navigation-drawer@1.3.0" has unmet peer dependency "react-native-gesture-handler@^1.0.12".
warning "react-navigation > react-navigation-drawer@1.3.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning "react-navigation > react-navigation-stack@1.4.0" has unmet peer dependency "react-native-gesture-handler@^1.0.0".
warning "react-navigation > react-navigation-stack@1.4.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning "react-navigation > react-navigation-tabs@1.1.4" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning "react-navigation > @react-navigation/native@3.5.0" has unmet peer dependency "react-native-gesture-handler@*".
warning " > reactotron-mst@3.1.2" has unmet peer dependency "reactotron-core-client@^2.5.0".
warning " > reactotron-mst@3.1.2" has incorrect peer dependency "mobx-state-tree@^1.4.0 || ^2.0.2".
warning " > @storybook/react-native@3.4.3" has unmet peer dependency "babel-runtime@>=6.0.0".
warning "@storybook/react-native > babel-loader@7.1.5" has incorrect peer dependency "babel-core@6".
warning " > jest-preset-ignite@0.6.1" has incorrect peer dependency "jest@^23.6".
error Command failed with exit code 1.

----------
⠴ ▸ installing react-native-gesture-handler@1.1.0 
      There were errors while generating the project. Run with --debug to see verbose output.
    
an error occured while installing ignite-bowser boilerplate.
{ Error: Command failed: yarn add react-native-gesture-handler@1.1.0
warning "react-navigation > react-navigation-drawer@1.3.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning "react-navigation > react-navigation-stack@1.4.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning "react-navigation > react-navigation-tabs@1.1.4" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning " > reactotron-mst@3.1.2" has unmet peer dependency "reactotron-core-client@^2.5.0".
warning " > reactotron-mst@3.1.2" has incorrect peer dependency "mobx-state-tree@^1.4.0 || ^2.0.2".
warning " > @storybook/react-native@3.4.3" has unmet peer dependency "babel-runtime@>=6.0.0".
warning "@storybook/react-native > babel-loader@7.1.5" has incorrect peer dependency "babel-core@6".
warning " > jest-preset-ignite@0.6.1" has incorrect peer dependency "jest@^23.6".
warning " > react-native-gesture-handler@1.1.0" has unmet peer dependency "react-art@>= 16.3.2".
warning " > react-native-gesture-handler@1.1.0" has unmet peer dependency "react-native-web@>= 0.7".
error Command failed with exit code 1.

    at ChildProcess.exithandler (child_process.js:294:12)
    at ChildProcess.emit (events.js:189:13)
    at maybeClose (internal/child_process.js:970:16)
    at Socket.stream.socket.on (internal/child_process.js:389:11)
    at Socket.emit (events.js:189:13)
    at Pipe._handle.close (net.js:597:12)
  killed: false,
  code: 1,
  signal: null,
  cmd: 'yarn add react-native-gesture-handler@1.1.0',
  stderr:
   'warning "react-navigation > react-navigation-drawer@1.3.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".\nwarning "react-navigation > react-navigation-stack@1.4.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".\nwarning "react-navigation > react-navigation-tabs@1.1.4" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".\nwarning " > reactotron-mst@3.1.2" has unmet peer dependency "reactotron-core-client@^2.5.0".\nwarning " > reactotron-mst@3.1.2" has incorrect peer dependency "mobx-state-tree@^1.4.0 || ^2.0.2".\nwarning " > @storybook/react-native@3.4.3" has unmet peer dependency "babel-runtime@>=6.0.0".\nwarning "@storybook/react-native > babel-loader@7.1.5" has incorrect peer dependency "babel-core@6".\nwarning " > jest-preset-ignite@0.6.1" has incorrect peer dependency "jest@^23.6".\nwarning " > react-native-gesture-handler@1.1.0" has unmet peer dependency "react-art@>= 16.3.2".\nwarning " > react-native-gesture-handler@1.1.0" has unmet peer dependency "react-native-web@>= 0.7".\nerror Command failed with exit code 1.\n' }
error installing boilerplate
```

I had CLT instead of XCode in my path, after changing this:

```
ignite new TheApp                                              

-----------------------------------------------
  (                  )   (                   
  )\ )   (        ( /(   )\ )    *   )       
 (()/(   )\ )     )\()) (()/(  ` )  /(   (   
  /(_)) (()/(    ((_)\   /(_))  ( )(_))  )\  
 (_))    /(_))_   _((_) (_))   (_(_())  ((_) 
 |_ _|  (_)) __| | \| | |_ _|  |_   _|  | __|
  | |     | (_ | | .` |  | |     | |    | _| 
 |___|     \___| |_|\_| |___|    |_|    |___|
-----------------------------------------------

An unfair headstart for your React Native apps.
https://infinite.red/ignite

-----------------------------------------------

🔥 igniting app TheApp
✔ Which boilerplate would you like to use? · ignite-bowser
✔ using the Infinite Red boilerplate v3 (code name 'Bowser')
✔ added React Native 0.59.3 in 23.87s
✔ Would you like to include Detox end-to-end tests? (y/N) · true

        You'll love Detox for testing your app! There are some additional requirements to
        install, so make sure to check out e2e/README.md in your generated app!
      

✖ ignite-bowser was not able to be installed. Is it a valid NPM module?
----------
Command failed: yarn add ignite-bowser --dev
warning @storybook/react-native > @storybook/core > autoprefixer > browserslist@2.11.3: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > autoprefixer > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-merge-rules > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-merge-rules > caniuse-api > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-minify-selectors > postcss-selector-parser > flatten@1.0.2: I wrote this module a very long time ago; you should use something else.
warning solidarity > gluegun > enquirer > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.
warning solidarity > gluegun > prompt-autocompletion > prompt-choices > set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.
warning solidarity > gluegun > enquirer > prompt-question > prompt-choices > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.
warning "react-navigation > react-navigation-drawer@1.3.0" has unmet peer dependency "react-native-gesture-handler@^1.0.12".
warning "react-navigation > react-navigation-drawer@1.3.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning "react-navigation > react-navigation-stack@1.4.0" has unmet peer dependency "react-native-gesture-handler@^1.0.0".
warning "react-navigation > react-navigation-stack@1.4.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning "react-navigation > react-navigation-tabs@1.1.4" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning "react-navigation > @react-navigation/native@3.5.0" has unmet peer dependency "react-native-gesture-handler@*".
warning " > reactotron-mst@3.1.2" has unmet peer dependency "reactotron-core-client@^2.5.0".
warning " > reactotron-mst@3.1.2" has incorrect peer dependency "mobx-state-tree@^1.4.0 || ^2.0.2".
warning " > @storybook/react-native@3.4.3" has unmet peer dependency "babel-runtime@>=6.0.0".
warning "@storybook/react-native > babel-loader@7.1.5" has incorrect peer dependency "babel-core@6".
warning " > jest-preset-ignite@0.6.1" has incorrect peer dependency "jest@^23.6".
error /Users/lukaszostrowski/projects/TheApp/TheApp/node_modules/postinstall-prepare: Command failed.
Exit code: 1
Command: node ./run.js
Arguments: 
Directory: /Users/lukaszostrowski/projects/TheApp/TheApp/node_modules/postinstall-prepare
Output:
npm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218590590-0.982510946613012/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.
npm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218590590-0.982510946613012/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

**ERROR** patch-package requires Git

  Please make sure that git is available in all of the necessary environments for your project.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! TheApp@0.0.1 patch: `patch-package`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the TheApp@0.0.1 patch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_49_51_924Z-debug.log
ERROR: "patch" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! TheApp@0.0.1 prepare: `npm-run-all patch hack:*`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the TheApp@0.0.1 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_49_51_947Z-debug.log
child_process.js:656
    throw err;
    ^

Error: Command failed: npm run prepare
npm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218590590-0.982510946613012/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.
npm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218590590-0.982510946613012/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

**ERROR** patch-package requires Git

  Please make sure that git is available in all of the necessary environments for your project.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! TheApp@0.0.1 patch: `patch-package`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the TheApp@0.0.1 patch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_49_51_924Z-debug.log
ERROR: "patch" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! TheApp@0.0.1 prepare: `npm-run-all patch hack:*`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the TheApp@0.0.1 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_49_51_947Z-debug.log

    at checkExecSyncError (child_process.js:616:11)
    at execSync (child_process.js:653:13)
    at Object.<anonymous> (/Users/lukaszostrowski/projects/TheApp/TheApp/node_modules/postinstall-prepare/run.js:14:5)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)

----------
⠏ ▸ installing react-native-gesture-handler@1.1.0 
      There were errors while generating the project. Run with --debug to see verbose output.
    
an error occured while installing ignite-bowser boilerplate.
{ Error: Command failed: yarn add react-native-gesture-handler@1.1.0
warning @storybook/react-native > @storybook/core > autoprefixer > browserslist@2.11.3: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > autoprefixer > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-merge-rules > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-merge-rules > caniuse-api > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-merge-rules > postcss-selector-parser > flatten@1.0.2: I wrote this module a very long time ago; you should use something else.
warning solidarity > gluegun > enquirer > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.
warning solidarity > gluegun > prompt-autocompletion > prompt-choices > set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.
warning solidarity > gluegun > enquirer > prompt-question > prompt-choices > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.
warning "react-navigation > react-navigation-drawer@1.3.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning "react-navigation > react-navigation-tabs@1.1.4" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning "react-navigation > react-navigation-stack@1.4.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".
warning " > reactotron-mst@3.1.2" has unmet peer dependency "reactotron-core-client@^2.5.0".
warning " > reactotron-mst@3.1.2" has incorrect peer dependency "mobx-state-tree@^1.4.0 || ^2.0.2".
warning " > @storybook/react-native@3.4.3" has unmet peer dependency "babel-runtime@>=6.0.0".
warning "@storybook/react-native > babel-loader@7.1.5" has incorrect peer dependency "babel-core@6".
warning " > jest-preset-ignite@0.6.1" has incorrect peer dependency "jest@^23.6".
warning " > react-native-gesture-handler@1.1.0" has unmet peer dependency "react-art@>= 16.3.2".
warning " > react-native-gesture-handler@1.1.0" has unmet peer dependency "react-native-web@>= 0.7".
error /Users/lukaszostrowski/projects/TheApp/TheApp/node_modules/postinstall-prepare: Command failed.
Exit code: 1
Command: node ./run.js
Arguments: 
Directory: /Users/lukaszostrowski/projects/TheApp/TheApp/node_modules/postinstall-prepare
Output:
npm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218628594-0.34179730759381366/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.
npm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218628594-0.34179730759381366/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

**ERROR** patch-package requires Git

  Please make sure that git is available in all of the necessary environments for your project.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! TheApp@0.0.1 patch: `patch-package`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the TheApp@0.0.1 patch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_50_30_685Z-debug.log
ERROR: "patch" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! TheApp@0.0.1 prepare: `npm-run-all patch hack:*`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the TheApp@0.0.1 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_50_30_709Z-debug.log
child_process.js:656
    throw err;
    ^

Error: Command failed: npm run prepare
npm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218628594-0.34179730759381366/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.
npm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218628594-0.34179730759381366/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

**ERROR** patch-package requires Git

  Please make sure that git is available in all of the necessary environments for your project.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! TheApp@0.0.1 patch: `patch-package`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the TheApp@0.0.1 patch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_50_30_685Z-debug.log
ERROR: "patch" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! TheApp@0.0.1 prepare: `npm-run-all patch hack:*`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the TheApp@0.0.1 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_50_30_709Z-debug.log

    at checkExecSyncError (child_process.js:616:11)
    at execSync (child_process.js:653:13)
    at Object.<anonymous> (/Users/lukaszostrowski/projects/TheApp/TheApp/node_modules/postinstall-prepare/run.js:14:5)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)

    at ChildProcess.exithandler (child_process.js:294:12)
    at ChildProcess.emit (events.js:189:13)
    at maybeClose (internal/child_process.js:970:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
  killed: false,
  code: 1,
  signal: null,
  cmd: 'yarn add react-native-gesture-handler@1.1.0',
  stderr:
   'warning @storybook/react-native > @storybook/core > autoprefixer > browserslist@2.11.3: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.\nwarning @storybook/react-native > @storybook/core > css-loader > cssnano > autoprefixer > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.\nwarning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-merge-rules > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.\nwarning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-merge-rules > caniuse-api > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.\nwarning @storybook/react-native > @storybook/core > css-loader > cssnano > postcss-merge-rules > postcss-selector-parser > flatten@1.0.2: I wrote this module a very long time ago; you should use something else.\nwarning solidarity > gluegun > enquirer > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.\nwarning solidarity > gluegun > prompt-autocompletion > prompt-choices > set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.\nwarning solidarity > gluegun > enquirer > prompt-question > prompt-choices > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.\nwarning "react-navigation > react-navigation-drawer@1.3.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".\nwarning "react-navigation > react-navigation-tabs@1.1.4" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".\nwarning "react-navigation > react-navigation-stack@1.4.0" has unmet peer dependency "react-native-screens@^1.0.0 || ^1.0.0-alpha".\nwarning " > reactotron-mst@3.1.2" has unmet peer dependency "reactotron-core-client@^2.5.0".\nwarning " > reactotron-mst@3.1.2" has incorrect peer dependency "mobx-state-tree@^1.4.0 || ^2.0.2".\nwarning " > @storybook/react-native@3.4.3" has unmet peer dependency "babel-runtime@>=6.0.0".\nwarning "@storybook/react-native > babel-loader@7.1.5" has incorrect peer dependency "babel-core@6".\nwarning " > jest-preset-ignite@0.6.1" has incorrect peer dependency "jest@^23.6".\nwarning " > react-native-gesture-handler@1.1.0" has unmet peer dependency "react-art@>= 16.3.2".\nwarning " > react-native-gesture-handler@1.1.0" has unmet peer dependency "react-native-web@>= 0.7".\nerror /Users/lukaszostrowski/projects/TheApp/TheApp/node_modules/postinstall-prepare: Command failed.\nExit code: 1\nCommand: node ./run.js\nArguments: \nDirectory: /Users/lukaszostrowski/projects/TheApp/TheApp/node_modules/postinstall-prepare\nOutput:\nnpm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218628594-0.34179730759381366/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.\nnpm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218628594-0.34179730759381366/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.\n\n**ERROR** patch-package requires Git\n\n  Please make sure that git is available in all of the necessary environments for your project.\n\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! TheApp@0.0.1 patch: `patch-package`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the TheApp@0.0.1 patch script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_50_30_685Z-debug.log\nERROR: "patch" exited with 1.\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! TheApp@0.0.1 prepare: `npm-run-all patch hack:*`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the TheApp@0.0.1 prepare script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_50_30_709Z-debug.log\nchild_process.js:656\n    throw err;\n    ^\n\nError: Command failed: npm run prepare\nnpm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218628594-0.34179730759381366/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.\nnpm WARN lifecycle The node binary used for scripts is /var/folders/yx/kggc1ff96zxgwn7p4kq2qwc40000gn/T/yarn--1561218628594-0.34179730759381366/node but npm is using /Users/lukaszostrowski/.nvm/versions/node/v10.15.3/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.\n\n**ERROR** patch-package requires Git\n\n  Please make sure that git is available in all of the necessary environments for your project.\n\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! TheApp@0.0.1 patch: `patch-package`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the TheApp@0.0.1 patch script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_50_30_685Z-debug.log\nERROR: "patch" exited with 1.\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! TheApp@0.0.1 prepare: `npm-run-all patch hack:*`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the TheApp@0.0.1 prepare script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /Users/lukaszostrowski/.npm/_logs/2019-06-22T15_50_30_709Z-debug.log\n\n    at checkExecSyncError (child_process.js:616:11)\n    at execSync (child_process.js:653:13)\n    at Object.<anonymous> (/Users/lukaszostrowski/projects/TheApp/TheApp/node_modules/postinstall-prepare/run.js:14:5)\n    at Module._compile (internal/modules/cjs/loader.js:701:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)\n    at Module.load (internal/modules/cjs/loader.js:600:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:531:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)\n    at startup (internal/bootstrap/node.js:283:19)\n' }
error installing boilerplate
```

--- a/src/cli/check.js
+++ b/src/cli/check.js
@@ -67,4 +67,13 @@ if (debugMode) {
   if (!yarn) {
     process.exit(exitCodes.INVALID_GLOBAL_DEPENDENCY)
   }
+  
+  var xCodeVersion = shell.exec('xcode-select -v',{silent:true}).stdout;
+  var xCodeInstalled = RegExp(/xcode-select version \d+/).test(xCodeVersion);
+  
+  if(!xCodeInstalled) {
+    console.log('We can\'t detect XCode installed on your machine. Check if XCode is installed and available in PATH');
+    process.exit(exitCodes.INVALID_GLOBAL_DEPENDENCY);
+  }
+  
 }

--- a/src/cli/check.js
+++ b/src/cli/check.js
@@ -68,12 +68,15 @@ if (debugMode) {
     process.exit(exitCodes.INVALID_GLOBAL_DEPENDENCY)
   }
   
-  var xCodeVersion = shell.exec('xcode-select -v',{silent:true}).stdout;
-  var xCodeInstalled = RegExp(/xcode-select version \d+/).test(xCodeVersion);
+  // If Mac OS, check if XCode is installed
+  if (process.platform === "darwin") {
+    var xCodeVersion = shell.exec('xcode-select -v',{silent:true}).stdout;
+    var xCodeInstalled = RegExp(/xcode-select version \d+/).test(xCodeVersion);
   
-  if(!xCodeInstalled) {
-    console.log('We can\'t detect XCode installed on your machine. Check if XCode is installed and available in PATH');
-    process.exit(exitCodes.INVALID_GLOBAL_DEPENDENCY);
+    if(!xCodeInstalled) {
+      console.log('We can\'t detect XCode installed on your machine. Check if XCode is installed and available in PATH');
+      process.exit(exitCodes.INVALID_GLOBAL_DEPENDENCY);
+    }
   }
   
 }

